### PR TITLE
Fix the center after the zoom

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: src/controllers/AbstractAppController.js [hunk 1]
```diff
@@ -171,6 +171,15 @@ export function AbstractAppController($scope, $injector, mobile) {
       this.options.map
     )
   );
+  // To fix the map position constraint, we need to set the center
+  let onModeEnd = false;
+  map.on('moveend', () => {
+    if (!onModeEnd) {
+      onModeEnd = true;
```
Comment (llienher) [inline src/controllers/AbstractAppController.js:178]: ModeEnd or MoveEnd ?

Comment (sbrunner) [reply src/controllers/AbstractAppController.js]: Ouops, thanks :-)

**Review by llienher — COMMENTED** [review]
Is it that the onMoveEnd is always false anyways so it makes the use of it not necessary ? If it is set to false as soon it is set to true, so it's always checks the condition.

If I undestand correctly, we would need something like that ?
`   map.on('moveend', () => view.setCenter(view.getCenter()));`

**Comment by sbrunner** [comment]
No, it's not always false this is to avoid infinite loop...

**Review by llienher — APPROVED** [review]

**Pull request merged**